extensions/eclipse-che-theia-remote-impl-k8s/src/node/k8s-workspace-service-impl.ts

```diff
@@ -71,8 +71,12 @@ export class K8sWorkspaceServiceImpl implements WorkspaceService {
   }
 
   public async getWorkspaceSettings(): Promise<WorkspaceSettings> {
-    console.log('workspaceService.getWorkspaceSettings() not supported');
-    return {};
+    console.log('workspaceService.getWorkspaceSettings() is implemented partially');
+
+    return {
+      CHE_PLUGIN_REGISTRY_URL: this.env.getPluginRegistryURL(),
```

Comment:
note that these variables are available in Che/DevWorkspaceOperator mode but not in pure DevWorkspace Operator mode so please check that there is no error on pure DevWorkspace

Reply:
then, how can we get links to plugin registry in pure DevWorkspace Operator mode?

Reply:
config map does not contain internal UR

![Screenshot from 2021-10-05 18-13-21](https://user-images.githubusercontent.com/1655894/136051576-824a5160-b85a-4f98-84ff-f74d50c12731.png)

Reply:
we could display the notification or use online instance of the registry at github.io